addon/addon/modifiers/sprite.ts

```diff
@@ -46,10 +59,21 @@ export default class SpriteModifier extends Modifier<SpriteModifierArgs> {
       );
       this.lastBounds = this.currentBounds;
       this.lastComputedStyle = this.currentComputedStyle;
-      this.currentBounds = getDocumentPosition(element);
+      this.currentBounds = getDocumentPosition(
+        element,
+        {
+          withAnimations: false,
+          playAnimations: false,
+        },
+        this.#animations
+      );
       this.currentComputedStyle = copyComputedStyle(element);
       this.alreadyTracked = true;
     }
+  }
+
+  finishSnapshot(): void {
+    this.#animations = [];
     once(this, this.clearTrackedPosition);
```

Comment:
I've added a "finishSnapshot" too as I saw multiple calls of `this.clearTrackedPosition` when many animations were happening because many captureSnapshots may cover multiple runloops.